libsolidity/formal/CHC.cpp

```diff
@@ -1254,6 +1297,13 @@ vector<smtutil::Expression> CHC::currentStateVariables(ContractDefinition const&
 	return applyMap(SMTEncoder::stateVariablesIncludingInheritedAndPrivate(_contract), [this](auto _var) { return currentValue(*_var); });
 }
 
+smtutil::Expression CHC::currentInitialConstraints(vector<VariableDeclaration const*> const& _vars) const
```

Comment:
About initial value of fold, I mean changing the signature to
```suggestion
smtutil::Expression CHC::currentInitialConstraints(vector<VariableDeclaration const*> const& _vars, smutil::Expression _initialValue) const
  return fold(_vars, _initialValue, [this](auto&& _conj, auto _var) {
```

And call it by  `conj = currentInitialContracts(stateVariables..., conj)`. Maybe not as good as I thought it would be, since we have to type out `conj` again.

Reply:
Ah I see what you mean